Test/New-WhiskeyProGetUniversalPackage.Tests.ps1

```diff
@@ -866,6 +879,17 @@ Describe 'New-WhiskeyProGetUniversalPackage.when cleaning' {
     Then7zipShouldNotExist
 }
 
+Describe 'New-WhiskeyProGetUniversalPackage.when initializing' {

```

Comment:
This test should be failing because the task's `ShouldInitialize` flag is not set to `$true`. Add assertions that:

* 7Zip was installed
* The task doesn't do anything but install 7zip. Currently, it copes files into a temp directory for packaging.

Reply:
Done